app/controllers/placement_reports_controller.rb

```diff
@@ -0,0 +1,15 @@
+class PlacementReportsController < ApplicationController
+  after_action :verify_authorized
+
+  def index
+    authorize :application, :see_reports_page?
+    placement_report = PlacementExportCsvService.new(casa_org_id: current_organization.id).perform
```

Comment:
How about we just pass the organization itself instead of the id?
```suggestion
    placement_report = PlacementExportCsvService.new(casa_org: current_organization).perform
```